tst/EnergyPlus/unit/HeatBalanceIntRadExchange.unit.cc

```diff
@@ -216,4 +221,91 @@ TEST_F(EnergyPlusFixture, HeatBalanceIntRadExchange_UpdateMovableInsulationFlagT
     EXPECT_TRUE(!DidMIChange);
 }
 
+TEST_F(EnergyPlusFixture, HeatBalanceIntRadExchange_AlignInputViewFactorsTest)
```

Comment:
Note the **COMPLETE** coverage for [HeatBalanceIntRadExchange::AlignInputViewFactors](http://energyplus.s3-website-us-east-1.amazonaws.com/coverage/2019-08-01-EnergyPlus-8229db616d-x86_64-gcc-7.4UTCD/EnergyPlus/HeatBalanceIntRadExchange.cc.gcov.html).